### DISCUSSION TITLE
Limit TDP on CPU

### DISCUSSION THREAD

**Discussion by kykym** [DavidS95/Smokeless_UMAF#9] [Q&A]
Hello, I have HP Omen 16 with Ryzen 5 5600H and Radeon RX 6600M. 

Can anyone tell me how to limit CPU's TDP in UMAF?

 Windows options aren't suitabe for me because any CPU's limitation causes power to be sucked off from the dGPU aswell

**Comment by DavidS95** [comment]
Limiting the cpu tdp would be the exact same as its done in windows using this tool, the windows app simply temporarily modifies the same value as this would. You would need to disable smartshift to change the behavior.

**Reply by kykym** [reply]
> Limiting the cpu tdp would be the exact same as its done in windows using this tool, the windows app simply temporarily modifies the same value as this would. You would need to disable smartshift to change the behavior.

I disabled it and the same behavior continued... like nothing happened - GPU was suffocating again, did I do it right?
![20230316_232635](https://user-images.githubusercontent.com/8003779/225830568-95f8fca2-cbc6-4a46-af8f-b7a88223810a.jpg)

**Reply by kykym** [reply]
I even changed the TDP here, In bios, but I'm under the impression that those values were ignored by the system.

**Reply by kykym** [reply]
Is this possible or I did something wrong here too?

**Comment by kykym** [comment]
I was wrong all the time, the TDP limit on the CPU is not a good option for me, I should've locked into proper undervolting instead

**Comment by Picco1990** [comment]
I want to increase the tdp, i have a notebook with ryzen 5500u 8GB 3200 MHZ single chanel w10 i installed AMD-APU-Tuning-Utility the program limits the power (eco mode works normally) but it cannot leave it above 15w, it goes up to about 25w but at the same time it drops and stabilizes at 15w. it is not giving thermal trotling, the highest temperature is 64ºC, I updated the chipset in an attempt to make it work but nothing happened, the bios is very simple and does not have options to increase the vram much less increase the tdp of the device, using Smokeless_UMAF i can change that?
